### PR TITLE
Add Workspaces endpoints

### PR DESCRIPTION
**Pull request by arturs105**

### CODE REVIEW

File: Toggl.Ultrawave/Network/Endpoints/WorkspacesEndpoints.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Toggl.Ultrawave.Network
+{
+    internal struct WorkspacesEndpoints
```
Comment (amulware) [inline Toggl.Ultrawave/Network/Endpoints/WorkspacesEndpoints.cs]: I think this reads nicer if it's just `WorkspaceEndpoints` without the plural s. Or is there a good case for the plural I'm missing?

Comment (amulware) [inline Toggl.Ultrawave/Network/Endpoints/WorkspacesEndpoints.cs]: (also ping @willsb so we'll decide this once for all future endpoint collections)

Comment (willsb) [inline Toggl.Ultrawave/Network/Endpoints/WorkspacesEndpoints.cs]: Singular, please, kthxbai

Comment (arturs105) [inline Toggl.Ultrawave/Network/Endpoints/WorkspacesEndpoints.cs]: Well, i looked at how clients were named - all, but one (`UserClient`) were named with plural - `WorkspacesClient`, `TasksClient` etc. So I thought endpoints should be named similarly.

But maybe You are right and we should stick to singular `WorkspaceEndpoints`

Comment (amulware) [inline Toggl.Ultrawave/Network/Endpoints/WorkspacesEndpoints.cs]: I think what makes this one sound funny to me is the double-plural. But yeah, we can always look over this again in the future and check for consistency. Not a big deal.

**Pull request merged**